query/filtering_lexer_test.go

```diff
@@ -37,6 +36,9 @@ func TestFilteringLexer(t *testing.T) {
 		LeToken{},
 		LeToken{},
 		NullToken{},
+		InsensitiveEqToken{},
```

Comment:
InToken is missing..

Reply:
Also good to have negative tests for empty array(`[]`), array with both strings and numbers(`["abc", 1, 2]`), unclosed array(`[1,2,`)